src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/sdo/SdoGeneratorService.java

```diff
@@ -368,6 +372,13 @@ private SdoDocumentFormSmall getTemplateDataSmall(CaseData caseData, String judg
                 SdoHelper.hasSmallClaimsVariable(caseData, "smallClaimsNumberOfWitnessesToggle")
             );
 
+        if (hasFlightDelayData && featureToggleService.isSdoR2Enabled()) {
+            sdoDocumentFormBuilder.smallClaimsFlightDelayToggle(
```

Comment:
Why we need this new variable "hasFlightDelayData" ?

Reply:
Think we dont need it now as we the smallClaimsFlightDelayToggle would control the showing/hiding of Flight delay section. hasFlightDelayData flag is just additional check to update flight delay details only when new template is called.
Can delete this flag check as its not needed anymore.